gunicorn/config.py

```diff
@@ -820,12 +820,12 @@ class PreloadApp(Setting):
 class Sendfile(Setting):
     name = "sendfile"
     section = "Server Mechanics"
-    cli = ["--sendfile"]
+    cli = ["--no-sendfile"]
     validator = validate_bool
-    action = "store_true"
+    action = "store_false"
     default = True
     desc = """\
-        Enables or disables the use of ``sendfile()``.
+        Disables the use of ``sendfile()``.
 
         .. versionadded:: 19.2
```

Comment:
Can you add a versionchanged directive and describe the change there. Something like:

```rst
.. versionchanged:: 19.4
   Foo bar
```

Also, perhaps we need to keep `--sendfile` as an alias to `--no-sendfile` for a few release.